tools/run_tests/build_artifact_python.sh

```diff
@@ -43,7 +43,11 @@ GRPC_PYTHON_USE_CUSTOM_BDIST=0  \
 GRPC_PYTHON_BUILD_WITH_CYTHON=1 \
 ${SETARCH_CMD} python setup.py  \
     bdist_wheel                 \
-    sdist                       \
+    sdist
```

Comment:
I would say that the "certain effect" that we seek is "orderly distribution of software written in Python, Cython, and C to the three major computing platforms of our time and with a not-crazy mix of source and precompiled assets". I think that we do care how we get there - we want to get their through orthodox, supported use of the distribution tools.

I think that that's what the authors and maintainers of the distribution tools want us to be doing, and every issue we file is a way of letting them know that there either isn't a supported, simple way of distributing our software, or isn't a way of finding the behavior we need from their tools.

I don't think that anyone will think poorly of us for filing issues saying "we want to accomplish X, we think we need Y, please assist". Maybe Y will get implemented or maybe we'll get redirected to some other part of the tool - but if we're right about the core of what we're doing (distributing our software) being hard, I don't think we're out of line and I think issues describing difficulty in using the distribution tools will be well received.

Why am I wrong and what am I missing?